### ISSUE TITLE
Heroku maxprocs: Leaving GOMAXPROCS=8: CPU quota undefined

### ISSUE THREAD

**Issue by IanVaughan** [DataDog/heroku-buildpack-datadog#234]
When running a bash console `heroku run bash` on either staging or production, I get this output:

```
Running bash on ⬢ my-app-staging... up, run.5756 (Hobby)
2021/01/19 12:06:47 maxprocs: Leaving GOMAXPROCS=8: CPU quota undefined

Running bash on ⬢ my-app-production... up, run.8380 (Standard-1X)
2021/01/19 12:04:15 maxprocs: Leaving GOMAXPROCS=8: CPU quota undefined
```

If I remove this buildpack then there is no log.

My app info is as follows:
```
=== staging
Addons:         deployhooks:http
                fixie:tricycle
                heroku-postgresql:hobby-basic
                heroku-redis:hobby-dev
                scout:chair
                sendgrid:starter
Auto Cert Mgmt: true
Dynos:          web: 1, worker: 1, run: 1
Region:         eu
Repo Size:      49 MB
Slug Size:      134 MB
Stack:          heroku-20

=== production
Addons:         deployhooks:http
                fixie:commuter
                heroku-postgresql:standard-2
                heroku-postgresql:standard-2
                heroku-redis:premium-0
                rollbar:essentials-30k
                scout:chair
                sendgrid:starter
Auto Cert Mgmt: true
Dynos:          web: 1, worker: 1, run: 1
Region:         eu
Repo Size:      28 MB
Slug Size:      246 MB
Stack:          heroku-20
```

I've not noticed any problems when  this line is logged, but it is curious.

**Comment by arapulido** [comment]
@IanVaughan Thanks for reporting! This is logged by one of the components that the agent uses, and it doesn't cause any harm. Feel free to ignore it. But this should be included in the agent logs, not outside of them. I will report this to the agent team to see if we can remove that log line.

**Comment by arapulido** [comment]
This started on agent `7.23`

**Comment by arapulido** [comment]
This is now fixed with the latest buildpack and agent 7.26.0
